populate/src/main/resources/seed/i18n/dev/languageTexts.json

```diff
@@ -16,9 +16,44 @@
   {"keyName": "taskTypeSurveys", "text": "DEV_Surveys", "language": "dev"},
   {"keyName": "taskTypeConsent", "text": "DEV_Consent", "language": "dev"},
   {"keyName": "taskTypeForms", "text": "DEV_Forms", "language": "dev"},
-  {"keyName": "taskStart", "text": "DEV_Start", "language": "dev"},
-  {"keyName": "taskContinue", "text": "DEV_Continue", "language": "dev"},
+  {"keyName": "start", "text": "DEV_Start", "language": "dev"},
+  {"keyName": "continue", "text": "DEV_Continue", "language": "dev"},
+  {"keyName": "save", "text": "DEV_Save", "language": "dev"},
   {"keyName": "outreachLearnMore", "text": "DEV_Learn More", "language": "dev"},
+  {"keyName": "cancel", "text": "DEV_Cancel", "language": "dev"},
+  {"keyName": "goBack", "text": "DEV_Go Back", "language": "dev"},
+  {"keyName": "name", "text": "DEV_Name", "language": "dev"},
+  {"keyName": "givenName", "text": "DEV_Given Name", "language": "dev"},
+  {"keyName": "familyName", "text": "DEV_Family Name", "language": "dev"},
+  {"keyName": "editName", "text": "DEV_Edit Name", "language": "dev"},
+  {"keyName": "editBirthDate", "text": "DEV_Edit Birthday", "language": "dev"},
+  {"keyName": "birthDate", "text": "DEV_Birthday", "language": "dev"},
+  {"keyName": "mailingAddress", "text": "DEV_Mailing Address", "language": "dev"},
+  {"keyName": "editMailingAddress", "text": "DEV_Edit Mailing Address", "language": "en"},
```

Comment:
```suggestion
  {"keyName": "editMailingAddress", "text": "DEV_Edit Mailing Address", "language": "dev"},
```

Reply:
great catches! not sure how those slipped through 🤔